src/tensors/indexmanipulations.jl

```diff
@@ -248,17 +248,31 @@ function twist!(t::AbstractTensorMap, i::Int; inv::Bool=false)
     end
     return t
 end
+function twist!(t::AbstractTensorMap, is; inv::Bool=false)
+    if !all(in(allind(t)), is)
+        msg = "Can't twist indices $is of a tensor with only $(numind(t)) indices."
+        throw(ArgumentError(msg))
+    end
+    (BraidingStyle(sectortype(t)) == Bosonic() || isempty(is)) && return t
+    N₁ = numout(t)
+    for (f₁, f₂) in fusiontrees(t)
+        θ = prod(i -> i <= N₁ ? twist(f₁.uncoupled[i]) : twist(f₂.uncoupled[i - N₁]), is)
+        inv && (θ = θ')
+        rmul!(t[f₁, f₂], θ)
+    end
+    return t
+end
 
 """
-    twist(t::AbstractTensorMap, i::Int; inv::Bool=false)
-        -> t
+    twist(tsrc::AbstractTensorMap, i::Int; inv::Bool=false) -> tdst
+    twist(tsrc::AbstractTensorMap, is; inv::Bool=false) -> tdst
 
-Apply a twist to the `i`th index of `t` and return the result as a new tensor.
+Apply a twist to the `i`th index of `tsrc` and return the result as a new tensor.
```

Comment:
I forgot to mention that the doc string needed an update to reflect the `is` argument here.